crates/fuel-core/src/schema/da_compressed.rs

```diff
@@ -0,0 +1,51 @@
+use super::{
+    scalars::HexString,
+    ReadViewProvider,
+};
+use crate::{
+    fuel_core_graphql_api::{
+        IntoApiResult,
+        QUERY_COSTS,
+    },
+    query::da_compressed::DaCompressedBlockData,
+    schema::scalars::U32,
+};
+use async_graphql::{
+    Context,
+    Object,
+};
+
+pub struct DaCompressedBlock {
+    bytes: Vec<u8>,
+}
+
+impl From<Vec<u8>> for DaCompressedBlock {
+    fn from(bytes: Vec<u8>) -> Self {
+        Self { bytes }
+    }
+}
+
+#[Object]
+impl DaCompressedBlock {
+    async fn bytes(&self) -> HexString {
+        HexString(self.bytes.clone())
+    }
```

Comment:
Not related to this PR, but I think `HexString` should take reference instead. Since cloning in some cases can be expensive. Plus I think we clone for transaction as well. What is not good. Would be nice to create an issue for it

Reply:
I'm not sure how that would work. The lifetime of hex string is tied to the lifetime of the original bytes?

Reply:
Yes, or `HexString` could create hexed string on construction instead of cloning and creating it after.